src/style.css

```diff
@@ -1,3 +1,64 @@
-body {
+/* stylelint-disable color-hex-case */
+/* stylelint-disable string-quotes */
+html {
+  font-family: 'Roboto', Helvetica, sans-serif;
+  font-size: 12px;
+}
+
+.body {
+  min-width: 1024px;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo__img {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
   margin: 0;
 }
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  text-decoration: none;
+  text-transform: uppercase;
+  cursor: pointer;
+  padding: 23px 0; /* як це прописати замість тупого підбору? */
```

Comment:
remove such comments - ask in the GH comments or even better in the chat :)
code should be clean

in the mockup you can check it (at least approximate value +- 1px):
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/2dfad953-c877-48fe-aae4-cb7ba13a331c)